source/whats-new.txt

```diff
@@ -33,11 +34,23 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.12.0:
```

Comment:
Q: I see that we have this for v11, but do we need it? I've only seen ".0" patch versions added if/when a ".1" patch is released to distinguish the patch versions.